### DISCUSSION TITLE
how to display withCount in table?

### DISCUSSION THREAD

**Discussion by tarikhagustia** [rappasoft/laravel-livewire-tables#839] [Q&A] [answered]
Hello,
i have following query
```
public function builder(): Builder
    {
        return ContactList::query()
            ->withCount('contacts')
            ->where('user_id', auth()->id()); // Select some things
    }
```
but when adding this line on column, this error appear
```
SQLSTATE[42703]: Undefined column: 7 ERROR: column contact_lists.contacts_count does not exist
```

how can i display `contacts_count` ?

**Comment by giles247** [comment, marked as answer]
Try this
` 
 Column::make('Contacts Count')
	->label(function ($row) {
		return $row->contacts_count;
	}),
`

**Reply by tarikhagustia** [reply]
thanks, its work!

**Reply by DanielGSoftware** [reply]
@redactuk works, but I wonder how you'd make this work when you also want to use search/enable? 😄

**Reply by giles247** [reply]
@DanielGSoftware Can't help you with that. I use this package in one project under long-term development, and since move to v2 pretty much all my previous methods of enabling search/sort are broken. I believe there is a least one PR related to broken sorting, so until some further fixes get done on this package I've not bothered to try and get examples like this to work for search/sort.

**Reply by DanielGSoftware** [reply]
Hmm I get that, we decided to upgrade to v2 one week ago. A lot of problems are arising during the process, and some functionality just doesn't work exist/work anymore, which I wonder why. Now I'm wondering if we should just stay at v1 for now as well.
